<launch>
  <arg name="simulation" default="False"/>
  <arg name="serial_device" default="/dev/ttyUSB0"/>
  <arg name="name_prefix" default="left_hand"/>

  <include file="$(find schunk_svh_driver)/launch/svh_controller.launch">
    <arg name="standalone" value="False" />
    <arg name="simulation" value="$(arg simulation)" />
    <arg name="serial_device" value="$(arg serial_device)" />
  </include>

  <node name="svh_sin_test" pkg="schunk_svh_driver" type="svh_sin_test" output="screen" >
    <remap from="~channel_targets" to="/svh_controller/channel_targets"/>
    <param name="name_prefix" value="$(arg name_prefix)" />
  </node>
  
</launch>
